gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/BundleMapperService.java

```diff
@@ -121,6 +128,8 @@ public Bundle mapToBundle(RCMRIN030000UK06Message xmlMessage) {
         var templates = templateMapper.mapResources(ehrExtract, patient, encounters, practiseCode);
         addEntries(bundle, templates);
 
+        mapDiagnosticReports(bundle, ehrExtract, patient, encounters, observations, observationComments, practiseCode);
+
         LOGGER.debug("Mapped Bundle with [{}] entries", bundle.getEntry().size());
```

Comment:
I'm sorry, it's not your change, but could you move this debug after `unknownPractitionerHandler.updateUnknownPractitionersRefs(bundle);`? As it can add additional practitioner entry.

Reply:
Sure, done :)